templates/Layout/DNRoot_environment.ss

```diff
@@ -10,22 +10,7 @@
 
 <div class="content">
 
-	<% with $CurrentEnvironment %>
```

Comment:
Why move this bit into the new extension point? That bit in here already contains an extension in fact :-)